Makefile

```diff
@@ -303,7 +303,7 @@ kind-extensions-up: $(KIND) $(KUBECTL)
 	REPO_ROOT=$(REPO_ROOT) ./hack/kind-extensions-up.sh
 kind-extensions-down: $(KIND)
```

Comment:
Seems like Kind can be removed here?

Reply:
Good point, cleaned up excess kind there.